src/controllers/bookingForm.js

```diff
@@ -0,0 +1,43 @@
+const nodemailer = require('nodemailer');
+require("env2")("/home/nareman/Desktop/mdf/Move-Dance-Feel/.env");
```

Comment:
This needs a little change. env2 can takes a path from where it is activated, so where the package.json is, so you can use "./.env" instead of "/home/nareman/Desktop/mdf/Move-Dance-Feel/.env" because that is specific to you.

Reply:
> This needs a little change. env2 can takes a path from where it is activated, so where the package.json is, so you can use "./.env" instead of "/home/nareman/Desktop/mdf/Move-Dance-Feel/.env". Afraid that is specific to you.

fixed it